main.go

```diff
@@ -196,6 +196,10 @@ func main() {
 	    arguments = append(arguments, "--trigger-http")
 	}
 
+	if (params.IngressSettings != "") {
+        arguments = append(arguments, "--ingress-settings", params.IngressSettings)
```

Comment:
Can you match indentation to the rest of the code?

Reply:
I did indented now using tabs as the most of the code is indented. 
To avoid such indentation issues it would be better to have spaces instead of tabs.